packages/mars/src/execute/sendTransaction.ts

```diff
@@ -41,6 +43,10 @@ export async function sendTransaction(
   }
   console.log()
 
+  if (logFile) {
+    fs.appendFileSync(logFile, `Transaction: ${name}\n  Hex data: ${tx.data}\n\n`)
```

Comment:
You should better extract it to a function. And it's worth adding a hash to a log. Also data should start from a new line, it will make it easier to copy when it's large